lib-content-scrapers/src/jvmMain/kotlin/com/ustadmobile/lib/contentscrapers/abztract/ScraperTypes.kt

```diff
@@ -1,7 +1,12 @@
 import com.ustadmobile.lib.contentscrapers.abztract.YoutubeChannelIndexer
 import com.ustadmobile.lib.contentscrapers.abztract.YoutubePlaylistIndexer
 import com.ustadmobile.lib.contentscrapers.abztract.YoutubeScraper
+import com.ustadmobile.lib.contentscrapers.apache.ApacheIndexer
```

Comment:
There is a spelling error in the package name: ab**z**tract instead of a**b**stract

Reply:
yes on purpose